### ISSUE TITLE
Cmd + click not working on macOS

### ISSUE THREAD

**Issue by Taitava** [Taitava/obsidian-shellcommands#347]
### Discussed in https://github.com/Taitava/obsidian-shellcommands/discussions/346

<div type='discussions-op-text'>

<sup>Originally posted by **luckman212** May  4, 2023</sup>
I just started playing with this amazing plugin. 

I wanted to try Cmd+click to redirect the output to one of my wrappers, but it doesn't seem to have any effect. The command still runs in its default context.

- Obsidian 1.2.8 (installer 1.2.7)
- macOS 13.3.1

<img width="748" alt="screenshot_AwRNevjM_gh" src="https://user-images.githubusercontent.com/1992842/236062877-aad0b5ca-c1c5-445b-b2e5-c4dd78532c9b.png">

</div>

- [ ] Edit 2023-06-25: Also need to ensure that Cmd + clicking on the link icon (![kuva](https://github.com/Taitava/obsidian-shellcommands/assets/13002142/03f1c77c-e1c4-430e-a9c9-b636609f70e7)) works, too.

**Comment by Taitava** [comment]
I haven't yet tried if this works, but there are three `event.ctrlKey` occurrences that could probably be fixed:

![kuva](https://github.com/Taitava/obsidian-shellcommands/assets/13002142/4c99e2d2-312b-4e5a-a29d-77fdd9a50ee6)

Create a function `isCmdOrCtrlPressed()` in `Common.ts`, next to `CmdOrCtrl()`:

```TypeScript
isCmdOrCtrlPressed(event: MouseEvent) {
    if (Platform.isMacOS) {
        return event.metaKey;
    } else {
        return event.ctrlKey;
    }
}
```

- [ ] Replace the three `event.ctrlKey` occurrences with `isCmdOrCtrlPressed(event)`. Then test on macOS that it really works.

**Comment by Taitava** [comment]
Fixed. Will be released in `0.20.1`.

**Comment by Taitava** [comment]
Released.
